pycbc/workflow/segment.py

```diff
@@ -850,10 +850,11 @@ def get_triggered_coherent_segment(workflow, out_dir, sciencesegs, tag=None):
                                            'on-before')))
     onafter = int(os.path.basename(cp.get('workflow-exttrig_segments',
                                           'on-after')))
-    padding = int(os.path.basename(cp.get('workflow-exttrig_segments',
-                                          'pad-data')))
+    padding = int(os.path.basename(cp.get('inspiral', 'pad-data')))
```

Comment:
I see no reason why this can't be given in the workflow-exttrig_segments section in the config file. Changed in https://github.com/a-r-williamson/pycbc/commit/cf671262776a9b8410ca88ecc1b445bc25d5d445